CHANGELOG.md

```diff
@@ -1,5 +1,144 @@
 # Version changelog
 
+## 0.21.0
+
+### New Features and Improvements
+* Fix get_workspace_client in GCP ([#532](https://github.com/databricks/databricks-sdk-py/pull/532)).
+* Use all-apis scope with external-browser ([#563](https://github.com/databricks/databricks-sdk-py/pull/563)).
+* Make a best effort attempt to initialise all Databricks globals ([#562](https://github.com/databricks/databricks-sdk-py/pull/562)).
+* Note: Backwards incompatible changes - Settings are now nested, please see the API changes below.
+
+### Documentation
+* Add Files API docs to the SDK Documentation ([#556](https://github.com/databricks/databricks-sdk-py/pull/556)).
+* New example to list compute resource for SUBMIT_RUN job runs ([#572](https://github.com/databricks/databricks-sdk-py/pull/572)).
+* Sort index pages by name in docs ([#560](https://github.com/databricks/databricks-sdk-py/pull/560)).
+* Add back enums to docs ([#557](https://github.com/databricks/databricks-sdk-py/pull/557)).
+
+### API Changes
+#### Added
+Services:
+- [w.permission_migration](https://databricks-sdk-py.readthedocs.io/en/latest/workspace/permission_migration.html) workspace-level service.
```

Comment:
shouldn't this be `w.settings.permission_migration`?

Reply:
synced offline but mentioning here for visibility: Not this one but for all the below services, they should be nested. Added them, there is a bug in changelog generation for nested services, raised it in slack, will create a ticket to track.